google/cloud/security/common/data_access/load_data_sql_provider.py

```diff
@@ -16,6 +16,7 @@
 
 # TODO: Investigate improving so we can avoid the pylint disable.
 # pylint: disable=line-too-long
+from google.cloud.security.common.data_access.csv_writer import GROUPS_FIELDNAMES
```

Comment:
I think Andrew has been suggesting importing the module instead of the class or variable.